open-banking-brasil-financial-api-1_ID1-ptbr.md

```diff
@@ -51,37 +51,37 @@ O Financial-grade API 1.0 do Open Banking Brasil consiste nas seguintes partes:
 
 Estas partes são destinados a serem usados com [RFC6749], [RFC6750], [RFC7636], [OIDC], [FAPI-1-Baseline] e [FAPI-1-Advanced]
 
-.# Introdução  {#Introduction}
+# Introdução  {#Introduction}
```

Comment:
This is done deliberately as it is how the XML to RFC tools work (or any other RFC) so please revert.